authentication/pipeline/user.py

```diff
@@ -125,12 +118,9 @@ def require_profile_update_user_via_saml(
         # No name information passed, skipping
         pass
 
-    profile, _ = Profile.objects.update_or_create(
-        user=user,
-        defaults={
-            'name': user.get_full_name()
-        },
-    )
+    profile = user.profile
```

Comment:
Registering a new user via SAML breaks at this point (there is no existing profile).

```
web_1      |   File "./authentication/pipeline/user.py", line 121, in require_profile_update_user_via_saml
web_1      |     profile = user.profile
web_1      |   File "/usr/local/lib/python3.6/site-packages/nplusone/core/signals.py", line 23, in wrapped
web_1      |     ret = func(*args, **kwargs)
web_1      |   File "/usr/local/lib/python3.6/site-packages/django/db/models/fields/related_descriptors.py", line 407, in __get__
web_1      |     self.related.get_accessor_name()
web_1      | django.db.models.fields.related_descriptors.RelatedObjectDoesNotExist: User has no profile.
```